lib/ZcapClient.js

```diff
@@ -0,0 +1,228 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+import {DEFAULT_HEADERS, httpClient} from '@digitalbazaar/http-client';
+import {SECURITY_CONTEXT_V2_URL} from 'jsonld-signatures';
+import {CapabilityDelegation} from 'ocapld';
+import {generateId} from 'bnid';
+import jsigs from 'jsonld-signatures';
+import {signCapabilityInvocation} from 'http-signature-zcap-invoke';
+const {Ed25519Signature2018} = jsigs.suites;
+
+export class ZcapClient {
+  /**
+   * An object that can make HTTPS requests such as node's https.Agent or ky.
```

Comment:
```suggestion
   * An object that can make HTTPS requests that include capability invocations.
```

Reply:
I was suggesting this change before I saw what this text was about -- since it's in the same block as the constructor text. The HttpsAgent is responsible for managing connection persistence and reuse.